source/renderer/app/api/voting/requests/createWalletSignature.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type { RequestConfig } from '../../common/types';
+import type { SignatureParams } from '../types';
+import { request } from '../../utils/requestBinary';
```

Comment:
Do we really need this `requestBinary`? We should rather use the existing `request` with modifications if they are needed.